azavea/cac-tripplanner
# Bugfix/kak/tour reorder highlighting#1212

## Overview

Fix tour map segment highlighting on hover after removing or reordering destinations.


### Notes

There was two separate issues: an off-by-one error in assigning order after reordering, and not handling changing the assigned order of subsequent places after removing one.


## Testing Instructions

 * Go to a tour map page
 * Remove a place by clicking the 'x' in the top right of its card
 * Hovering over places should still highlight the correct map segment
 * Reorder places by dragging a card
 * Hovering over places should still highlight the correct map segment
 * Set and/or clear the origin
 * Hovering over places should still highlight the correct map segment

Closes #1212
